data/config/items/dungeons/stronghold-of-security/boots.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rs:Fancy boots": {
```

Comment:
Standard `underscore_case` is requested for all config ids ❤️

Reply:
Will do! Thanks for the heads up.